README.md

```diff
@@ -1 +1,51 @@
-# dat
+# Delta Acceptance Testing (DAT)
+
+The DAT project makes it easy to generate Delta Lake reference tables and run integration tests to check that different implementations of the Delta Lake PROTOCOL behave as expected.
+
+The Delta Lake implementations can clone this repo, generate the reference tables, and implement the suggested integration tests in the language of their choice.
+
+## Installation
+
+You can install the dependencies for this project by running `poetry install`.  Run `poetry shell` to activate the virtual environment.    If `poetry shell` does not activate the environment on your machine, you may need to run this command:
+
+```
+source "$( poetry env list --full-path | grep Activated | cut -d' ' -f1 )/bin/activate"
+```
+
+Run `python -m dat.main` to make sure everything is working properly.  You should see instructions on how to use DAT, including the commands you can run.
+
+## How to build tables
+
+Start by removing the `out` directory with `rm -rf out`.
+
+Create the reference tables with `python -m dat.main write-generated-reference-tables`.
+
+This will create reference tables as follows:
+
+```
+out/
+  tables/
+    generated/
+      reference_table_1/
+        delta/
+          ...
+        parquet/
+          table_content.parquet
+        table_metadata.json
+      reference_table_2/
+        ...
+```
+
+## Example integration tests
```

Comment:
this is an "example" but where is the "protocol". what do we do with the reference tables? what is the delta table? What is the parquet table?

An example is useful to illustrate how to implement the protocol ... but we need to specify that first.

Reply:
@MrPowers friendly ping

Reply:
I added `assert_metadata_equality(delta_log_path, expected_metadata_path)`.  The `assert_metadata_equality` method doesn't exist yet, but we can hope it will someday, haha.  It'll compare the actual `_delta_log` to the expected log in `table_metadata.json` and make sure they have the same list of transactions.